src/serverenvironment.cpp

```diff
@@ -1817,10 +1818,19 @@ void ServerEnvironment::getSelectedActiveObjects(
 			current_raw_normal = current_normal;
 		}
 		if (collision) {
+			PointabilityType pointable;
+			LuaEntitySAO* lsao = dynamic_cast<LuaEntitySAO*>(obj);
+			if (lsao && pointabilities) {
+				pointable = pointabilities->matchObject(lsao->getName(),
+						usao->getArmorGroups()).value_or(props->pointable);
+			} else {
+				pointable = props->pointable;
+			}
```

Comment:
Before this I do a dynamic cast to LuaEntitySAO and a LuaEntitySAO can never be a PlayerSAO, so the name can never be a playername.

The hierarchy looks like this:
ActiveObject -> ServerActiveObject -> UnitSAO -> LuaEntitySAO
ActiveObject -> ServerActiveObject -> UnitSAO -> PlayerSAO

You probably meant the client environment and not the serverenvironment.cpp, since there I need to check this.
It is fixed now.

Reply:
Oh sorry, I misunderstand this.
It's fixed now and works properly with players.